client/src/utils/getValueFiatToDisplay.ts

```diff
@@ -56,5 +60,9 @@ export default function getValueFiatToDisplay({
     displayCurrency === 'jpy' ? 0 : 2,
   );
 
+  if (valueFiat === '0.00' && showLessThanZero) {
+    return `< ${prefix}0.00`;
```

Comment:
In JPY (Japanese Yen) there are no subdivisions like cents, so an exception needs to be handled here.

![image](https://github.com/golemfoundation/octant/assets/25887234/1248c52b-f3ef-4021-a776-a592078d3bf4)